src/main.ts

```diff
@@ -28,11 +28,12 @@ async function bootstrap() {
 
     app.useGlobalPipes(new ValidationPipe());
     const NODE_ENV = app.get(AppConfigService).nodeEnv;
+    const LATEST_RELEASE = app.get(AppConfigService).latestReleaseLink;
     if (NODE_ENV !== "production") {
         const config = new DocumentBuilder()
             .setTitle("Chingu Dashboard Project")
             .setDescription(
-                "Chingu Dashboard API<br> default access: logged in (user)",
+                `Chingu Dashboard API <a href='${LATEST_RELEASE}'> Version: ${LATEST_RELEASE.split("tag/")[1]} (Latest-dev) </a> <br> default access: logged in (user)`,
             )
             .setVersion("1.0")
```

Comment:
I think this should be updated too.

Reply:
it crashed when the env is not set, need to put a `?`

Reply:
I relase the link we have to use is `https://github.com/chingu-x/chingu-dashboard-be/releases/tag/1.1.0-alpha` I thought it would be something more generic like `https://api.github.com/repos/chingu-x/chingu-dashboard-be/releases`

I don't think this will work it would be a pain to update all the env variables in all the deployments for every release.

Also, this is not a "secret" so shouldn't need to be in env. I guess you put it in env because you were thinking to have a different value in each environment?

Originally I was thinking to just hardcode it on every release, but I think a better solution is getting the version from package.json and we'd just make sure it's in sync with the github release version (we can add this step to the release procedure)

Another thing is I also don't want it to show it being the "latest" version for all the deployments.